geomstats/_backend/tensorflow/__init__.py

```diff
@@ -74,15 +141,15 @@ def get_mask_float(indices, mask_shape):
     return tf_mask
 
 
-def duplicate_array(x, n_samples, axis=0):
+def _duplicate_array(x, n_samples, axis=0):
     multiples = _np.ones(ndim(x) + 1, dtype=_np.int32)
     multiples[axis] = n_samples
     return tile(to_ndarray(x, ndim(x) + 1), multiples)
 
 
-def get_vectorized_mask_float(
+def _get_vectorized_mask_float(
```

Comment:
Maybe coordinate with @pchauchat on this? :/

Reply:
We discussed it yesterday, I had some issues with Git so I had to keep my renaming. From what I understood, the important part was to add the underscore, since the function is to be used only internally. Then, whether we call it (vectorized_)mask_from_indices or get_(vectorized_)mask_float is another question.

Reply:
Sure! Right now in the code base it is `mask_from_indices`, so let's do `_mask_from_indices`? Unless you have strong preference for one of the two other namings?